pkgs/tools/networking/telepresence/default.nix

```diff
@@ -11,19 +11,19 @@ let
       leaveDotGit = true;
```

Comment:
This line is actually evil. Why is it required?

Reply:
Seems to be unnecessary